src/com/core/tictactoe/Board.java

```diff
@@ -80,6 +76,72 @@ String getPassivePlayerSign() {
         return activePlayerSign.equals(FIRST_PLAYER_SIGN) ? SECOND_PLAYER_SIGN : FIRST_PLAYER_SIGN;
     }
 
+    String[][] getRowsInBoard() {
+        return this.createArrayOfLines(this.places);
+    }
+
+    private String[][] getAllLines() {
+        ArrayList<String[]> lines = new ArrayList<>();
+
+        String[][] rowsInBoard = getRowsInBoard();
+        lines.addAll(Arrays.asList(rowsInBoard));
+
+        String[][] columnsInBoard = getColumnsInBoard(rowsInBoard);
+        lines.addAll(Arrays.asList(columnsInBoard));
+
+        lines.add(this.getTopLeftDiagonal(rowsInBoard));
```

Comment:
These two could become #getDiagonalsInBoard to be consistent with getRows/getColumns